### ISSUE TITLE
Can't connect to audible 

### ISSUE THREAD

**Issue by masterdot** [openaudible/openaudible#803]
Hey there,

I can't create a working connection. The entered credentials of both accounts are correct, but after logging in there is a error message that says the page requested is missing.
When I do select the browse menu option my library is shown correctly. But it still said it's not connected.
If you need anything, just tell me :)

**Comment by openaudible** [comment]
Did you try logging in from the Controls menu: Connect to Audible?

**Comment by masterdot** [comment]
Yes, from the menu and via browser to check if the credentials do work. BTW, german login page. But I couldn't connect to .com either... Didn't have any problems before and it's the most actual version.

**Comment by openaudible** [comment]
Should be working. Can you quit and restart OA, attempt a "connect to audible", and if connected, do a full rescan.. Then send a bug report from the About menu with issue 803 listed. I'll be able to see any error messages and get a better sense of what happened.

**Comment by masterdot** [comment]
Made the requested steps. Hope you'll find the problems... Did work perfectly before :)

**Comment by openaudible** [comment]
Ugh. Looks like a bug in the built in web browser.  Ugh. Not sure what could have changed except perhaps the underlying Linux. Sorry it may take a while to figure that one out.

**Comment by deheingit** [comment]
Hi, same issue here with the docker image. 
Looking fwd. to a fix ;)

**Comment by openaudible** [comment]
Spent some time digging into the WebKit browser library and looking at switching to chrome.. but it is non-trivial. Not sure exactly what broke..

**Comment by masterdot** [comment]
Sorry for the work. Maybe you can add a different browser with the app? Don't know how the docker stuff works, but maybe it's an idea?

**Comment by openaudible** [comment]
No problem. Several possible fixes are under investigation... Will just take time.

This bug pretty much only affects logging into your audible. So, you could run OA on a windows or mac, log in, and copy the credentials.json from the mac to your linux machine.. And it should show you "connected".

**Comment by deheingit** [comment]
thanks, copying the json worked ;)
Best
 Dennis

**Comment by lolmc** [comment]
Hi, where is the json file located on a Mac so I can copy it to my GNU/Linux install?

**Comment by lolmc** [comment]
I've copied MacOS credentials.json to my Ubuntu machine but the browser isn't rendering pages correctly - some of page is ok but mostly just blank screen and I am unable to complete sign in. The progress bar at the base of the browser window remains at the beginning and doesn't move.
I am still unable to authenticate. Do you have any idea when this may be fixed?

**Comment by Davidsmithdevelopment** [comment]
Having the same issue - sounds like you have it ironed out, but sent the bug through the app just in case.

I don't have a Mac, and am unable to find the credentials.json  file on my pc. Any guidance would be appreciated

**Comment by openaudible** [comment]
To find your credentials.json file, go to an OpenAudible that is connected to audible on Mac or PC. Should show "Connected: Yes"

Go to Preferences, and click on "Show" button on Preferences under "Directories": (3rd row, near top of prefs.) In that directory is the credentials.json which you can copy to the new location, found by repeating the above on the linux machine.

We're exploring adding a chrome browser to the app to improve the browser.. but that too has a lot of known/unknown issues.

**Comment by sirkubax** [comment]
Same here, Login Browser open, but it hangs, sometimes Connect button get's grey, Sometimes it reloads, ask to fill some challange, but picture is inactive, and website irresponsive...

**Comment by openaudible** [comment]
OK! New fix for linux has been added to 3.4. It downloads a mini-chrome browser for the login.
Let me know if you see any problems!

**Comment by lolmc** [comment]
Hi, thanks for your efforts to fix this - worked first time.

**Comment by OnkelTem** [comment]
Hey guys, I cannot connect.
I downloaded OA on Kubuntu 20.04 as the AppImage.
When I run it and then go to Controls > Connect to Audible I get a popup browser window with login.
There I log in using my email and password, but afterwards it's just showing the same page:

![image](https://user-images.githubusercontent.com/114060/156951949-487f0041-4d14-4773-8df3-2bf87202d899.png)

I tried it several times. Tried restarting the app.

Any ideas?
